src/renderer/components/ft-input/ft-input.js

```diff
@@ -285,6 +285,10 @@ export default defineComponent({
       this.$refs.input.focus()
     },
 
+    blur() {
+      this.$refs.input.focus()
```

Comment:
Did you mean to call focus here?

Reply:
No 🙈, forgot to change it after copying the method above. Fixed it now.